docker-compose.yml

```diff
@@ -38,6 +38,7 @@ services:
     labels:
       com.centurylinklabs.watchtower.enable: '${WATCHTOWER_ENABLED:-false}'
     environment:
+      - RLN_CONTRACT_REPO_COMMIT=${RLN_CONTRACT_REPO_COMMIT:-master}
```

Comment:
For me it seems safer to default here to the last known working commit than latest master.
WDYT?

Reply:
Thanks, I agree that we should use a commit that we know works instead of master/main. I changed it in https://github.com/waku-org/waku-simulator/pull/95/commits/c118e37dc2eb72f0b164a480810456ab5878666b